storage/drivers/template.go

```diff
@@ -1,52 +0,0 @@
-package graphdriver
```

Comment:
(Unrelated: It turns out there are ~no non-trivial implementations of `CreateFromTemplate` now, so that method could be deleted and the caller simplified a bit not to compute its extra parameters. But that’s way out of scope.)